tests/fixtures/mod.rs

```diff
@@ -76,3 +81,85 @@ pub fn tmpdir() -> TempDir {
 pub fn port() -> u16 {
     free_local_port().expect("Couldn't find a free local port")
 }
+
+/// Run miniserve as a server; Start with a temporary directory, a free port and some
+/// optional arguments then wait for a while for the server setup to complete.
+#[fixture(args=&[] as &[&str])]
```

Comment:
Could you switch this to the [modern way](https://docs.rs/rstest/0.11.0/rstest/attr.fixture.html#default-values)?